src/language-server/fileSet.ts

```diff
@@ -30,13 +28,6 @@ export class FileSet {
     this.excludes = excludes;
   }
 
-  pushIncludes(files: string[]) {
-    this.includes.push(...files);
-  }
-  pushExcludes(files: string[]) {
-    this.excludes.push(...files);
-  }
```

Comment:
These are not used anymore since `fileSet` has been split into `GraphQLProject.configFileSet` and `InternalGraphQLProject.fileSet`